### PR TITLE
[: add German translation

### PR DESCRIPTION
**Pull request by ZweiundzwanzigSiebtel**
- [ yes] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [ yes] The page has 8 or fewer examples.
- [yes ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [yes ] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [yes ] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.de/common/[.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# [
+
+> Teste und vergleiche Dateien und Variablen.
+> Gibt 0 zurück, falls der Ausdruck als wahr ausgewertet wird und 1 falls der Ausdruck zu falsch ausgewertet wird.
```
Comment (marchersimon) [inline pages.de/common/[.md:4]: What do you think about this?
```suggestion
> Gibt 0 zurück, wenn der Ausdruck wahr ist und 1 wenn nicht.
```

File: pages.de/common/[.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+> Weitere Informationen: <https://www.gnu.org/software/coreutils/test>.
+
+- Überprüfe die übergebene Variable und den übergebenen String auf Gleichheit:
```
Comment (marchersimon) [inline pages.de/common/[.md:7]: That reads a bit easier in my opinion
```suggestion
- Überprüfe, ob die gegebene Variable und der gegebene String gleich sind:
```

File: pages.de/common/[.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+`[ "{{$VARIABLE}}" == "{{/bin/zsh}}" ]`
+
+- Ermittle, ob die übergebene Variable leer ist:
```
Comment (marchersimon) [inline pages.de/common/[.md:11]:
```suggestion
- Ermittle, ob die gegebene Variable leer ist:
```

File: pages.de/common/[.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+`[ -z "{{$GIT_BRANCH}}" ]`
+
+- Ermittle, ob eine Datei existiert:
+
+`[ -f "{{pfad/zu/datei}}" ]`
+
+- Ermittle, ob ein Verzeichnis nicht leer ist:
```
Comment (marchersimon) [inline pages.de/common/[.md:19]: Shouldn't that be
```suggestion
- Ermittle, ob ein Verzeichnis nicht existiert:
```

File: pages.de/common/[.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+`[ ! -d "{{pfad/zum/verzeichnis}}" ]`
```
Comment (marchersimon) [inline pages.de/common/[.md:21]:
```suggestion
`[ ! -d "{{pfad/zu/verzeichnis}}" ]`
```

File: pages.de/common/[.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+
+- If-else Abfrage:
+
+`[ {{Ausdruck}} ] && {{echo "wahr"}} || {{echo "falsch"}}`
```
Comment (marchersimon) [inline pages.de/common/[.md:25]: Apart from a few exceptions, all arguments should be lowercase. Even in German.
```suggestion
`[ {{ausdruck}} ] && {{echo "wahr"}} || {{echo "falsch"}}`
```

**Comment by marchersimon** [comment]
@ZweiundzwanzigSiebtel if you need any help using GitHub just tell us.

**Review by marchersimon — APPROVED** [review]
Thanks @ZweiundzwanzigSiebtel

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**